# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/grep/unobsolete-egrep.patch
# Copyright (C) 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- grep-3.8/src/egrep.sh.vanilla	2022-09-05 12:22:12.531251775 +0200
+++ grep-3.8/src/egrep.sh	2022-09-05 12:22:17.048251506 +0200
@@ -1,4 +1,4 @@
 #!@SHELL@
 cmd=${0##*/}
-echo "$cmd: warning: $cmd is obsolescent; using @grep@ @option@" >&2
+#echo "$cmd: warning: $cmd is obsolescent; using @grep@ @option@" >&2
 exec @grep@ @option@ "$@"
